code/datums/diseases/advance/symptoms/cough.dm

```diff
@@ -18,23 +18,55 @@ BONUS
 /datum/symptom/cough
 
 	name = "Cough"
+	desc = "The virus irritates the throat of the host, causing occasional coughing."
 	stealth = -1
 	resistance = 3
 	stage_speed = 1
 	transmittable = 2
 	level = 1
 	severity = 1
+	base_message_chance = 15
+	symptom_delay_min = 2
+	symptom_delay_max = 15
+	var/infective = FALSE
+	threshold_desc = "<b>Resistance 3:</b> Host will drop small items when coughing.<br>\
+					  <b>Resistance 10:</b> Occasionally causes coughing fits that stun the host.<br>\
+					  <b>Stage Speed 6:</b> Increases cough frequency.<br>\
+					  <b>If Airborne:</b> Coughing will infect bystanders.<br>\
+					  <b>Stealth 4:</b> The symptom remains hidden until active."
+
+/datum/symptom/cough/Start(datum/disease/advance/A)
+	if(A.properties["stealth"] >= 4)
+		suppress_warning = TRUE
+	if(A.spread_flags &= AIRBORNE) //infect bystanders
+		infective = TRUE
+	if(A.properties["resistance"] >= 3) //strong enough to drop items
+		power = 1.5
+	if(A.properties["resistance"] >= 10) //strong enough to stun (rarely)
+		power = 2
+	if(A.properties["stage_rate"] >= 6) //cough more often
+		symptom_delay_max = 10
 
 /datum/symptom/cough/Activate(datum/disease/advance/A)
-	..()
-	if(prob(SYMPTOM_ACTIVATION_PROB))
-		var/mob/living/M = A.affected_mob
-		switch(A.stage)
-			if(1, 2, 3)
+	if(!..())
+		return
+	var/mob/living/M = A.affected_mob
+	switch(A.stage)
+		if(1, 2, 3)
+			if(prob(base_message_chance) && !suppress_warning)
```

Comment:
real nitpick that doesn't matter much here but more as a optimization lesson.
Switch the two around. Have a variable check first before an operation. It can save some processing time. Here it doesn't matter much but it can be handy to keep in mind for later.